### PR TITLE
Converted stellar-anchor-server into a reusable django app

### PR DESCRIPTION
**Pull request by JakeUrban**
As the title states, this PR makes all the changes necessary to use this codebase as a reusable django app. Part 2 of this project will be to refactor stellar-anchor-server to use this app (PR coming soon).

Functionally, this app should be no different from the previous version. I figured it would make more sense to wait until after the refactor to implement the request for defined integration points. A design proposal for these integration points is also coming soon.

See the README.md for installation and setup instructions.

### CODE REVIEW

**Comment by msfeldstein** [comment]
Should we have an minimal `example` application that shows how to use this properly?

**Comment by msfeldstein** [comment]
when you say `Define REPO_DIR in settings.py. Polaris uses this to find the .env file` do you mean in the main applications settings.py, or in polaris's settings.py?

File: README.md [hunk 1]
```diff
@@ -1,64 +1 @@
...
-1. Run celery: `$ pipenv run celery worker --app app --beat --workdir src -l info`
-1. Run a script to stream withdrawal transactions from Horizon: `$ pipenv run python src/manage.py watch_transactions`
-1. Run the project: `$ pipenv run python src/manage.py runserver`
-
-## Using the admin panel
-Through Django's admin panel, you can create assets, monitor transaction status, and do other administrative tasks.
-The above instructions for "Running the project locally" include the creation of an administrative user.
-Once the project is running locally, navigate to `https://localhost:8000/admin` in a browser. Enter the username and password you set for the superuser above. You should then see the admin panel.
-To create an asset, click `+ Add` in the `Assets` row of the `INFO` table. You can then edit the fields of an asset (its name, deposit values, withdrawal values) and save it.
+A SEP-24-compliant resuable django app.
```
Comment (msfeldstein) [inline README.md]: can you put the instructions you have in the PR in this file?

**Comment by JakeUrban** [comment]
@msfeldstein in step with this change, [stellar-anchor-server](https://github.com/stellar/stellar-anchor-server/tree/master) will become that minimal django project as an example of how to use this package.

Regarding `REPO_DIR`, I've clarified the instructions in the `README.md`.

File: polaris/polaris/models.py [hunk 2]
```diff
@@ -1,12 +1,43 @@
-"""This module defines the models for the transaction app."""
+"""This module defines the models for the info app."""
```
Comment (debnil) [inline polaris/polaris/models.py]: s/info/Polaris

**Pull request merged**